app/scripts/views/sign_in.js

```diff
@@ -4,26 +4,28 @@
 
 define([
   'cocktail',
+  'lib/auth-errors',
+  'lib/constants',
   'lib/promise',
+  'lib/session',
+  'stache!templates/sign_in',
   'views/base',
   'views/form',
-  'stache!templates/sign_in',
```

Comment:
I did this under the impression that we were trying to move towards alphabetical order everywhere; this list was a bit of a mess and it wasn't clear where the correct insertion point for the new dependency (`'views/mixins/inter-tab-channel-mixin'`) was otherwise. Hope that's okay.